### DISCUSSION TITLE
What is "Blink+ Plan"?

### DISCUSSION THREAD

**Discussion by jaclu** [blinksh/blink#1545] [Q&A] [answered]
I really enjoy this app as by far the best iOS terminal app, and I would love to support the team further!

I am currently on the Blink Classic Plan.

However before signing up for the Blink+ plan, it would seem reasonable to get some kind of list of added features planned or intended, even if nothing is decided, you should be able to tell us something about what you are selling :)

I could find nothing on your website, or by several searches.

To sell something and be totally evasive of what you are selling, at least to me seems a bit suspicious. I don't doubt your good intentions in your specific case, so I guess it is just oversight. But I guess I am not the only one wondering what this is meant to be/become.

Ideally add some pages on your website describing the different plans. Even if things might change, at least that would help me make more of a decision based on reasoning, and not only based on a vague sentence in Settings - Subscriptions :)

**Comment by carloscabanero** [comment, marked as answer]
Moving this to Discussions as I think it is a better fit here. 

So first of all, there is nothing sketchy behind the lack of information. We had to pull the trigger earlier than expected due to some organizational issues, and decided to go with what we had. We can do a much better job at this and we will. We are now working in one last **big** feature to complete the picture and I'm confident that will clear things out.

I cannot open our masterplan now, but, let me drop some clues. Ideas are always welcome :)

- Blink Plus already gives you access, without interruptions, to the leading terminal for iOS, the best and most complete Mosh and SSH support, a Code integration that is now a favorite editor of thousands of users, and the most advanced remote files integration (super performant sftp, Code to remote servers, and Files.app integration).
- We do not have enough with just being a terminal anymore, and we will be doing a lot to go beyond. Plus will include _early feature access_ before making it to the other versions.
- More advanced features to improve the connectivity of your device. Think what Mosh does for your shell, but applied to other areas too (tunnels, dev environments, etc…).
- We have been working on features that required our own infrastructure, like a better agent, better notifications, etc...
- Syncing Blink through multiple devices and multiple platforms. Including sensitive information like keys and other collaborative settings.
- Access to other cloud environments that require special authentication (AWS, Azure, GCE, etc…).
- ...

This is a big plan and will require more resources. I think we have already proved that we can be relevant and take things beyond where they stand right now. If we get enough people on board, our plan is to make the current version of Blink free. And that is what we are working for.
